src/scheduler/node_info.c

```diff
@@ -2884,7 +2885,9 @@ eval_simple_selspec(status *policy, chunk *chk, node_info **pninfo_arr,
 	int		cur_ncpus = 0;		/* current number of ncpus left to license */
 	int		licenses_allocated = 0;	/* licenses allocated to a job on a node */
 	resource_req	*ncpusreq = NULL;
+	resource_req	*aoereq = NULL;
 	resdef		*ncpusdef = find_resdef(consres, "ncpus");
+	resdef		*aoedef = find_resdef(allres, "aoe");
```

Comment:
Some compilers might complain that there are multiple variables being declared and initialized with independent calls to find_resdef().

Reply:
Also, this is unnecessary.  The point of getallres() is so you don't have to search by string for certain commonly used builtins.  AOE is one of them.  Use getallres(RES_AOE) instead of this.

Reply:
> Some compilers might complain that there are multiple variables being declared and initialized with independent calls to find_resdef().

Done.

>  The point of getallres() is so you don't have to search by string for certain commonly used builtins. AOE is one of them. Use getallres(RES_AOE) instead of this.
Done